index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Prakash Sakari">
+    <meta name="description"
+        content="Explore The Right Fit's Latest Collection Of Footwear For Men, Women, Boys, Girls & Senior Citizen. Fashionnable, Trendy, Classic and Comfortable Footwear for all.">
+    <title>Home | The Right Fit </title>
+    <link rel="stylesheet" href="https://uilight.netlify.app/main.css">
+    <link rel="stylesheet" href="main.css">
+</head>
+
+<body>
+    <header class="heading d-flex grow1-shrink1-basisauto align-center">
+        <div class="heading-title-icon d-flex align-center">
+            <img class="icon mr-1 border-radius-50" src="assets/The Right Fit-logos.jpeg" alt="lightbul">
+            <h1 class="heading-title">
+                <a class="link" href="/">The Right Fit</a>
+            </h1>
+        </div>
+        <div class="search-box-container relative">
+            <input class="search-box padding-all-8 border-radius-4" type="text" placeholder="Search">
+            <img src="/assets/outline_search_black_24dp.png" alt="" class="search-icon absolute left-0 top-0">
```

Comment:
Please provide "alt" description for accessibility.

Reply:
thanks, updated!